builder/FindGlobals.cmake

```diff
@@ -185,3 +183,52 @@ endfunction()
 function(report_add_target var target)
   set(${ARGV0} ${${ARGV0}} ${ARGV1} CACHE INTERNAL "" FORCE)
 endfunction()
+
+# Defined OS name and version and build info and build commit
+if(APPLE)
+  set(MFX_SYSTEM "MAC")
+elseif(UNIX)
+  execute_process(
+    COMMAND lsb_release -i
+    OUTPUT_VARIABLE OUTPUT
+    ERROR_VARIABLE ERROR
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+  )
+  if(NOT ERROR)
+    string(REPLACE  "Distributor ID:	" ""  MFX_LINUX_NAME "${OUTPUT}")
+    set(MFX_SYSTEM "${MFX_LINUX_NAME}")
+  endif()
+elseif(WIN32)
```

Comment:
WIN32 looks suspicious... Please, double check how to distinguish Linux.

Reply:
This is correct check.
https://cmake.org/cmake/help/v3.0/variable/WIN32.html